### PR TITLE
BUGFIX: `flow config:show` allow path with dots

### PR DESCRIPTION
**Pull request by mhsdesign**
related: #1903

**Upgrade instructions**

**Review instructions**

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked wit `!!!` and have upgrade-instructions

### CODE REVIEW

File: Neos.Utility.Arrays/Classes/ArrayIndexParser.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+     * foo[bar.buz].bing
+     *
+     * foo[bar.buz][bar.buz]
+     */
+    public static function parseFromString(string $input): array
+    {
+        $result = [];
+        $previousToken = null;
+        foreach (self::tokensFromString($input) as $token) {
+            switch ($token::class) {
```
Comment (mhsdesign) [inline Neos.Utility.Arrays/Classes/ArrayIndexParser.php:34]: ups this is php8

**Comment by mhsdesign** [comment]
it was discussed that we want to not overcomplicate the handling of such things but use json instead:
```
./flow configuration:show --path '["Node.Type:Lol", "properties"]'
```

configurationManager::get should also allow array syntax

**Comment by sorenmalling** [comment]
@mhsdesign What is the corresponding issue, that this PR solves?

**Comment by mhsdesign** [comment]
@sorenmalling there is none, its all just wip ;)

i was trying to fix https://github.com/neos/flow-development-collection/pull/1903#issuecomment-1184134424

**Comment by mhsdesign** [comment]
it was discussed that this might actually not be that use full after all especially when https://github.com/neos/neos-development-collection/pull/4091 gets introduced.

**Comment by mhsdesign** [comment]
Im happy how https://github.com/neos/neos-development-collection/pull/4091 and currently don’t see the need to allow escaping dots in paths, as this introduces unnecessary complexity

**Pull request closed without merging**